lib/terminus.js

```diff
@@ -27,7 +27,10 @@ function sendSuccess (res, info) {
   res.end(SUCCESS_RESPONSE)
 }
 
-function sendFailure (res, error) {
+async function sendFailure (res, error, onSendFailureDuringShutdown) {
```

Comment:
it seems like the params have started to get out of hand - should we convert to an object, so the order won't matter?

something like this:

```
async function sendFailure ({res, error, onSendFailureDuringShutdown}) 
```

Reply:
Agreed, converting...